server/routes/api.js

```diff
@@ -115,31 +119,40 @@ routes.get('/pairing/paired', (req, res) => {
   });
 });
 
-routes.get('/wishlist/:familyId', (req, res) => {
+routes.patch('/wishlist/:familyId/addItem', wishlistController.addItem);
+
+routes.patch('/wishlist/:familyId/create', wishlistController.create);
```

Comment:
consider making this a `PUT` to imply creation of a new object.
> The HTTP PUT method only allows complete replacement of a document. Unlike PUT, PATCH is not idempotent, meaning successive identical patch requests may have different effects. However, it is possible to issue PATCH requests in such a way as to be idempotent.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH